smoketests/__main__.py

```diff
@@ -80,10 +83,10 @@ def main():
         except FileNotFoundError:
             pass
         try:
-            os.symlink("spacetimedb-update", SPACETIME_BIN)
+            os.symlink("spacetimedb-update" + exe_suffix, SPACETIME_BIN)
         except OSError:
             import shutil
-            shutil.copyfile(SPACETIME_BIN.with_name("spacetimedb-update"), SPACETIME_BIN)
+            shutil.copyfile(SPACETIME_BIN.with_name("spacetimedb-update" + exe_suffix), SPACETIME_BIN)
```

Comment:
could we maybe group together the `"spacetimedb-update" + exe_suffix` into a `spacetimedb_update_bin` variable or something so we don't have to do this in multiple places?